README.md

```diff
@@ -4,10 +4,31 @@ Shot is an Android project you can use to write screenshot for your apps in a si
 
 ## What is this?
 
-``Shot`` is a Gradle plugin and a core android library thought to run screenshot tests for Android using the [screenshot testing Facebook SDK](http://facebook.github.io/screenshot-tests-for-android/) under the hood. This project provides a handy interface named ``ScreenshotTest`` and a ready to use ``ShotTestRunner`` you can use in order to simplify your tests design.
+``Shot`` is a Gradle plugin and a core android library thought to run screenshot tests for Android. This project provides a handy interface named ``ScreenshotTest`` and a ready to use ``ShotTestRunner`` you can use in order to write tests like these:
```

Comment:
Do you want to put emphasis on jetpack compose?

Reply:
We will need to support regular UI views for at least 2 or 3 years, so I left the details related to compose to the next paragraph 😃